importApp/src/main/java/com/commercetools/dataimport/joyrideavailability/AvailabilityJobConfiguration.java

```diff
@@ -0,0 +1,246 @@
+
+    @Bean
+    public Step setPricesStep(final BlockingSphereClient sphereClient,
+                              final ItemWriter<ProductUpdateCommand> setPriceWriter) {
+        final StepBuilder stepBuilder = stepBuilderFactory.get("createInventoryEntryStep");
+        return stepBuilder
+                .<Product, ProductUpdateCommand>chunk(productsImportStepChunkSize)
+                .reader(ItemReaderFactory.sortedByIdQueryReader(sphereClient, ProductQuery.of()))
+                .processor(setPriceProcessor(sphereClient))
+                .writer(setPriceWriter)
+                .build();
+    }
+
+    @Bean
+    @StepScope
+    private ItemReader<TypeDraft> customTypeReader(@Value("#{jobParameters['typesResource']}") final Resource typesJsonResource) throws IOException {
+        logger.info("URL_Types: " + typesJsonResource);
+        final ObjectReader reader = SphereJsonUtils.newObjectMapper().readerFor(new TypeReference<List<TypeDraft>>() { });
+        final InputStream inputStream = typesJsonResource.getInputStream();
+        final List<TypeDraft> list = reader.readValue(inputStream);
+        return new ListItemReader<>(list);
+    }
+
+    @Bean
+    protected ItemWriter<TypeDraft> customTypeWriter(final BlockingSphereClient sphereClient) {
+        return items -> items.forEach(customType -> sphereClient.executeBlocking(TypeCreateCommand.of(customType)));
+    }
+
+    @Bean
+    @StepScope
+    private ItemReader<ChannelDraftDsl> channelsDraftReader(@Value("#{jobParameters['channelsResource']}") final Resource channelsJsonResource) throws IOException {
+        logger.info("URL_Channels: " + channelsJsonResource);
+        final ObjectReader reader = SphereJsonUtils.newObjectMapper().readerFor(new TypeReference<List<ChannelDraftDsl>>() {
+        });
+        final InputStream inputStream = channelsJsonResource.getInputStream();
+        final List<ChannelDraftDsl> list = reader.readValue(inputStream);
+        return new ListItemReader<>(list);
+    }
+
+    @Bean
+    protected ItemWriter<ChannelDraftDsl> channelsDraftWriter(final BlockingSphereClient sphereClient) {
+        return items -> items.forEach(channelDraft -> sphereClient.executeBlocking(ChannelCreateCommand.of(channelDraft)));
+    }
+
+    @Bean
+    protected ItemProcessor<Product, List<InventoryEntryDraft>> inventoryEntryProcessor(final BlockingSphereClient sphereClient) {
+        return item -> {
+            getChannelsWithInventory(sphereClient);
+            return inventoryEntryListByChannel(item);
+        };
+    }
+
+    @Bean
+    public ItemWriter<List<InventoryEntryDraft>> inventoryEntryWriter(final BlockingSphereClient sphereClient) {
+        return entryLists -> entryLists.forEach(items -> items.forEach(inventoryEntry -> sphereClient.executeBlocking(InventoryEntryCreateCommand.of(inventoryEntry))));
+    }
+
+    @Bean
+    protected ItemProcessor<Product, ProductUpdateCommand> setPriceProcessor(final BlockingSphereClient sphereClient) {
```

Comment:
this is not a good method name to create a bean of the type ItemProcessor, since setPriceProcessor sounds like a Java bean setter